bigbluebutton-html5/imports/startup/client/logger.js

```diff
@@ -16,7 +16,7 @@ import { nameFromLevel } from '@browser-bunyan/levels';
 // externalURL is the end-point that logs will be sent to
 // Call the logger by doing a function call with the level name, I.e, logger.warn('Hi on warn')
 
-const LOG_CONFIG = Meteor.settings.public.clientLog || [{ target: 'console', level: 'info' }];
+const LOG_CONFIG = Meteor.settings.public.clientLog || { console: { enabled: true, level: 'info' } };
```

Comment:
You could move the default value to another variable called something like `LOG_CONFIG_DEFAULT`